src/components/GamePage/GamePage.jsx

```diff
@@ -8,15 +8,20 @@ import Header from '../Header/Header'
 import { gamePageSelector } from './GamePage.selector'
 
 function GamePage() {
+    let history = useHistory()
     const { id } = useParams()
     const { allGames } = useSelector(gamePageSelector)
     const game = allGames.find(item => item.id === id)
-
+    console.log(game)
     return (
         <div>
-            <Header isMainPage={false} />
+            <Header isMainPage={false} title={game.displayName} />
             <div className={styles['frame-container']}>
-                <iframe id={id} src={game.url} />
+                {game ? (
+                    <iframe id={id} src={game.url || 'https://wiki.com'} />
+                ) : (
+                    () => history.push('/game/original_dice_game')
```

Comment:
shouldn't we get this ID from the server?

Reply:
no) if we don't have game we run to dice